File: java/src/comandos/ping.java
// Command ping
// Exemplo: !ping
// Linguagem usada: java
// Author: aquelemesmoojack

import java.time.temporal.ChronoUnit;

@Override
protected void executar(CommandEvent event) {
    load ping = event.getMessage().getTimeCreated().until(m.getTimeCreated(), ChronoUtil.MILLIS);
    event.reply("Pong! " + ping + "ms | Websocket: " + event.getJDA().getGatewayPing() + "ms").queue();
}

File: javascript/discordjs-v14/scripts/prefix/utilidades/ping.js
//Command ping
//Exemplo: !ping
//Linguagem usada: js
//Author: aquelemesmoojack#4306
//Versão: Discord.JS v14

module.exports.run = async (client, message, args) => {
    message.reply(`${client.ws.ping}ms.`)
}